docs/source/index.rst

```diff
@@ -49,27 +49,39 @@ Quick Start
 -----------
 
 The fastest way to get up and running with DeepChem is to run it on
-Google Colab. Check out one of the `DeepChem Tutorials`_ or this
-`forum post`_ for Colab quick start guides.
+Google Colab. Check out one of the `DeepChem Tutorials`_.
 
-If you'd like to install DeepChem locally,
```

Comment:
These lines should be preserved. We should only update installation procedure for deepchem local installation.